applications/luci-app-https_dns_proxy/Makefile

```diff
@@ -3,14 +3,14 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_LICENSE:=GPL-3.0+
+PKG_LICENSE:=GPL-3.0-or-later
 PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
 
 LUCI_TITLE:=HTTPS DNS Proxy Web UI
 LUCI_DESCRIPTION:=Provides Web UI for HTTPS DNS Proxy
-LUCI_DEPENDS:=+luci-compat +luci-mod-admin-full +https_dns_proxy
```

Comment:
@stangri - I think you need to readd the `luci-compat` dependency, this is required if you use server side CBI rendering (like `Map()` or `Form()` in Lua files).

Reply:
@jow- does the dependency on `luci-mod-admin-full` not satisfy `luci-compat`?

Reply:
@jow- could you please address the Q above as I may have to urgently send PRs for my other luci apps.

Reply:
As I can see the dependency in the master branch is always `+luci-compat` and `+luci-mod-admin-full`